tests/cpp/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@
 # Copyright (c) 2013-2015, EPFL/Blue Brain Project
 #                     Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 #                     Raphael Dumusc <raphael.dumusc@epfl.ch>
+#
+# Change this number when adding tests to force a CMake run: 0
```

Comment:
In practice you don't do clean builds. You pull and make. And if you would do 'make ServerTests', you'll get an error :) Not in this commit though, as I touched another cmake file as well.